aiohttp/streams.py

```diff
@@ -635,10 +632,10 @@ def __init__(
         self._protocol = protocol
         self._limit = limit * 2
 
-    def feed_data(self, data: _T, size: int = 0) -> None:
-        super().feed_data(data, size)
+    def feed_data(self, data: _T) -> None:
+        super().feed_data(data)
 
-        if self._size > self._limit and not self._protocol._reading_paused:
+        if self._size < self._limit and not self._protocol._reading_paused:
```

Comment:
Why is this changed? Surely we don't want to pause reading if the size is still less than the limit...

Reply:
Under this condition, for some reason, the `test_feed_pause` test began falled